keyboards/hhkb/ansi/config.h

```diff
@@ -15,25 +15,21 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CONFIG_H
+#pragma once
 #define CONFIG_H
```

Comment:
```suggestion
```
Nuke this and the other similar lines, I couldn't highlight two lines at once in the previous comments/suggestions.

Reply:
Done. I should have noticed, thanks.